From 049d49a6c02e5546525fc1471fe5cf1449bd8977 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Mon, 7 Dec 2015 17:00:35 +0200
Subject: [PATCH 08/38] ARM: i.MX6UL: update defconfig for cl-som-imx6ul

Update defconfig for cl-som-imx6ul.
Solves:
1) a systemd serial console init hang at boot.
2) wlan0 "no wireless extensions."

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm/configs/cl_som_imx6ul_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/configs/cl_som_imx6ul_defconfig b/arch/arm/configs/cl_som_imx6ul_defconfig
index 3b9d475..357f013 100644
--- a/arch/arm/configs/cl_som_imx6ul_defconfig
+++ b/arch/arm/configs/cl_som_imx6ul_defconfig
@@ -1,6 +1,7 @@
 CONFIG_LOCALVERSION="-6UL_ga"
 CONFIG_KERNEL_LZO=y
 CONFIG_SYSVIPC=y
+CONFIG_FHANDLE=y
 CONFIG_NO_HZ=y
 CONFIG_HIGH_RES_TIMERS=y
 CONFIG_IKCONFIG=y
@@ -93,6 +94,7 @@ CONFIG_BT_HCIBCM203X=y
 CONFIG_BT_ATH3K=y
 CONFIG_BT_WILINK=m
 CONFIG_CFG80211=y
+CONFIG_CFG80211_WEXT=y
 CONFIG_MAC80211=y
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
-- 
1.9.1

